provider/spark.go

```diff
@@ -1459,15 +1461,9 @@ func (d *DatabricksExecutor) GetDFArgs(outputURI string, code string, sources []
 }
 
 func (spark *SparkOfflineStore) GetTransformationTable(id ResourceID) (TransformationTable, error) {
-	spark.Logger.Debugw("Getting transformation table", "ResourceID", id)
 	transformationPath := spark.Store.PathWithPrefix(fileStoreResourcePath(id), false)
-	transformationExactPath, err := spark.Store.NewestFileOfType(transformationPath, Parquet)
```

Comment:
Are we not getting the newest file anymore?

Reply:
I think this function is a bit confusing and I had the same thought when first viewing it.  This function is what's responsible for fetching a TF's source (at least in the offline store)

In the coordinator, it's used to get the source of a transformation when registering a label/feature. Its output is what you see when you go to the .txt file for a label or feature. By omitting the latest file lookup here and delegating it over to feature/training set evaluation (below) we ensure the latest is fetched during evaluation.

I thought about renaming it to getTransformationSource (but I don't think that's exactly how it's used for the SQL online store and I'm punting doing any refactoring rn)